mod_reforged/hooks/config/strings.nut

```diff
@@ -2126,7 +2126,8 @@ foreach (vanillaDesc in vanillaDescriptions)
 		Effects = [{
 			Type = ::UPD.EffectType.Passive,
 			Description = [
-				"When you equip a weapon whose perk group you have access to, gain the first, second, or third perk in that weapon\'s perk group if you have the corresponding perk in another weapon\'s perk group, regardless of its tier."
+				"When you equip a [Non-Hybrid Weapon|Concept.HybridWeapons] whose perk group you have access to, gain the first, second, and third perk in that weapon\'s perk group if you have the corresponding perk in another weapon\'s perk group, regardless of its tier."
+				"When equipping a [Hybrid Weapon|Concept.HybridWeapons] gain the first, second, and third perk in the alphabetically first weapon type of the hybrid weapon if you have the corresponding perk in a weapon perk group the [Hybrid Weapon|Concept.HybridWeapons] doesn't have. If you have the first and second perk from one of the [Hybrid Weapon\'s|Concept.HybridWeapons] perk groups, gain the corresponding first and second perk of the other perk group. Perks given by Weaponmaster do not qualify for this effect."
```

Comment:
This is the actual effect, that I implemented for Hybrid weapons (using your wording)

`"When you equip a [Hybrid Weapon|Concept.HybridWeapons] whose perk group you have access to, gain the first and second perk in that weapon\'s perk group if you have the corresponding perk in another weapon\'s perk group, regardless of its tier."`

 plus mention about the alphabetically-ness of it. But I don't know how that part works or is supposed to work. I didn't change that at all. I just made the third perk no longer being granted to hybrid weapons
 
It doesn't matter whether that perk comes from one of the hybrid weapons perk groups or from a perk group on the outside

Reply:
Okay, I think that's acceptable as well.